js/search/config.js.in

```diff
@@ -1 +0,0 @@
-const PKGDATADIR = '%pkgdatadir%';
```

Comment:
This is removed from the do_subst variable in the makefile, and other files still need pkgdatadir substituted.